src/components/Header.jsx

```diff
@@ -1,24 +1,37 @@
 import React from 'react';
+import { Navbar, Nav, NavItem } from 'react-bootstrap';
 import { Link } from 'react-router-dom';
 import cfgLogo from '../static/CfGLogoWhite.png';
 import routes from './pages/routes';
 
 const Header = () => (
-  <div className="header">
-    <div className="header__title">
-      <Link to="/">GoVoteGSO</Link>
-    </div>
-    <div className="header__details">
-      <div className="header__details-item">
+  <Navbar collapseOnSelect fluid className="header">
+    <Navbar.Header className="header__brand">
+      <Navbar.Brand className="header__title">
+        <Link to="/">GoVoteGSO</Link>
+      </Navbar.Brand>
+      <Navbar.Toggle />
+    </Navbar.Header>
+    <Navbar.Collapse>
+      <Nav pullRight>
         {
-          routes.map((route, index) => <Link to={route.to} key={index} className="btn btn-link header__details-action">{route.label}</Link>)
+          routes.map((route, index) => {
```

Comment:
The Travis is reporting the following error: 
```
./src/components/Header.jsx
  Line 18:  Unexpected block statement surrounding arrow body  arrow-body-style
```
Try doing an implicit return without the `return` statement

Thanks!